lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -169,4 +176,15 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
       ie -> if InformedEntity.parent_station?(ie), do: ie.stop
     end)
   end
+
+  defp has_nearby_redundancy?(%Alert{
+         informed_entities: [%{facility: %{id: informed_facility_id}} | _]
+       }) do
+    if data = @elevator_redundancy_data[informed_facility_id] do
+      data["nearby_redundancy?"]
+    else
+      Logger.error("Elevator #{informed_facility_id} does not exist in redundancy data")
```

Comment:
We're unlikely to see this unless we're actively checking for it, and the effect of an elevator incorrectly _not_ being filtered out is subtle enough that we maybe shouldn't count on it being reported. A Sentry report would be more attention-grabbing.

Reply:
My idea was to report to Sentry but didn't think about calling `Sentry.capture_message`. Can change Sentry reporting only.

Reply:
`Logger.error` currently doesn't report to Sentry; the "Logger integration" is for capturing process crashes only, by default. Capturing log _messages_ and reporting them to Sentry requires [enabling an option](https://docs.sentry.io/platforms/elixir/configuration/logging/#capture-log-messages). Ultimately I don't think it would be a bad idea to enable this: we'd be able to just `Logger.[error/warning]` and get console output in development, Sentry reports in production.

Reply:
Ahh ok not sure I realized that for some reason. Thank you.